docs/tutorial/t-setup-environment.md

```diff
@@ -0,0 +1,69 @@
+# Environment Setup
+
+This is part of the [Charmed Kafka Tutorial](/t/). Please refer to this page for more information and the overview of the content. 
+
+## Requirements
+
+For production environments, it is recommended to deploy at least 5 nodes for Zookeeper and 3 for Kafka. While the following requirements are meant to be for production, the charm can be deployed in much smaller environments.
```

Comment:
If we are in the tutorial, it is not useful to list "prod" requirements. 

I would revised these requirements to what needed for the tutorial, and then place the Prod requirements, into a separated section in "Reference/requirements.md"